osidb/serializer.py

```diff
@@ -687,8 +688,12 @@ def create(self, validated_data):
 
         # create new tracker instance in the local DB first
         # so we can create the links before the backend sync
-        # alerts will be created after links
-        tracker.save(no_alerts=True)
+        # alerts will be created after links and timestamp
+        # should be generated by BTS and will be fixed during sync
```

Comment:
Nitpick: Is there an extra word in the comment?

Reply:
Message improved, thanks.